### PR TITLE
Added menu items, with some functionality

### PR DESCRIPTION
**Pull request by TurnbulA**
Menu buttons have some functionality. When clicked first two buttons show which filter/ sort is applied 
Search button takes in data using enter key or button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,121 @@
       <section class="todo-list-header">
         <h1>To do list</h1>
       </section>
+      <section class="todo-list-menu">
```
Comment (mali891) [inline index.html]: These class names are quite confusing, let's catch up on Monday about them

Comment (TurnbulA) [reply index.html]: ok

File: todos.js [hunk 2]
```diff
@@ -45,3 +45,87 @@ const deleteEvent = () => {
...
+  dropButton.addEventListener("click", () => {
+    const activeContent = document.querySelector(
+      ".c-filter-option-active-text"
+    );
+    const reset = document.querySelector(".c-filter-option-active--reset");
+    if (dropButton.classList.contains("todo-list-menu--drop-complete")) {
+      activeContent.classList.add("c-filter-option-active-left");
+      document.querySelector(".c-filter-option-active-left").innerHTML =
+        "Filter: Completed items";
+      reset.style.display = "block";
```
Comment (mali891) [inline todos.js]: Toggle classes rather than inline properties

**Comment by mali891** [comment]
I just pulled the branch, good job so far @TurnbulA . I think some of the JS can be written a little bit cleaner, and it looks like there are some minor bugs in the UI. Let's catch up on Monday and go through them

File: index.html [hunk 3]
```diff
@@ -13,19 +13,19 @@
   </head>
   <body>
     <div class="container">
-      <section class="todo-list-header">
-        <h1>To do list</h1>
+      <section class="c-header">
```
Comment (stephen-algeo) [inline index.html:16]: this must have been a huge PITA to retro fit, well done for not wrecking it.

File: index.html [hunk 4]
```diff
@@ -13,19 +13,19 @@
...
+        <h1 class="c-header-heading">To do list</h1v b>
```
Comment (mali891) [inline index.html]: `</h1v b>` ?

File: index.html [hunk 5]
```diff
@@ -13,19 +13,19 @@
...
       </section>
-      <section class="todo-list-menu">
-        <div class="c-filter-option todo-list-menu__dropdown-filter">
-          <button class="c-filter-option__dropbtn c-filter-option__filter">
+      <section class="c-menu">
+        <div class="c-menu-control c-menu-item--filter">
```
Comment (mali891) [inline index.html]: There's a few modifiers in here (`--`), I'm not convinced there should be. Is there a good reason to have them in?

File: index.html [hunk 6]
```diff
@@ -36,29 +36,29 @@ <h1>To do list</h1>
...
-              class="todo-list-menu--drop-item todo-list-menu--drop-incomplete"
+              class="c-menu-control__dropdown--item c-menu-control__dropdown--incomplete"
             >
               Incomplete items
             </div>
           </div>
-          <div class="c-filter-option-active">
-            <p class="c-filter-option-active-text"></p>
-            <button class="c-filter-option-active--reset">
+          <div class="c-menu-control-active">
```
Comment (mali891) [inline index.html]: `--active` could be a modifier

File: index.html [hunk 7]
```diff
@@ -36,29 +36,29 @@ <h1>To do list</h1>
...
+            <p class="c-menu-control-active-text"></p>
```
Comment (mali891) [inline index.html]: Could there be a better name for this?

File: index.html [hunk 8]
```diff
@@ -84,7 +82,9 @@ <h1 class="c-header-heading">To do list</h1v b>
               />
             </svg>
           </button>
-          <div class="c-menu-control__dropdown--content c-menu-control__dropdown-sort">
+          <div
+            class="c-menu-control__dropdown--content c-menu-control__dropdown-sort"
```
Comment (mali891) [inline index.html]: modifier

File: index.html [hunk 9]
```diff
@@ -50,11 +50,9 @@ <h1 class="c-header-heading">To do list</h1v b>
...
-            <p class="c-menu-control-active-text"></p>
-            <button class="c-menu-control-active--reset">
-              Reset filter
-            </button>
+          <div class="c-menu-control--active">
+            <p class="c-menu-control--active-text"></p>
+            <button class="c-menu-control--active-reset"></button>
           </div>
         </div>
         <div class="c-menu-control c-menu-item--sort">
```
Comment (mali891) [inline index.html]: modifier

**Review by mali891 — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -9,130 +9,136 @@
...
+                  id="Path_3"
+                  data-name="Path 3"
+                  d="M12.07,28.758a.7.7,0,0,0-.995,0L6.843,32.99V8.907a.707.707,0,0,0-1.415,0V32.99L1.2,28.758a.7.7,0,0,0-.995.995l5.434,5.434a.708.708,0,0,0,1,0l5.434-5.434A.717.717,0,0,0,12.07,28.758Z"
+                  transform="translate(0 -8.2)"
+                  fill="#34495e"
+                />
               </svg>
             </button>
-          </form>
+            <div class="c-menu__dropdown-content c-menu__dropdown-sort">
```
Comment (mali891) [inline index.html]: These could probably be `ul` > `li`

File: index.html [hunk 11]
```diff
@@ -9,130 +9,136 @@
...
+                      fill="#e8e8e8"
+                    />
+                  </g>
+                </svg>
+              </button>
+            </form>
+          </div>
+        </div>
+        <div class="c-menu-status">
+          <div class="c-menu-status--active">
```
Comment (mali891) [inline index.html]: How did you come up with the name for this?

**Review by mali891 — APPROVED** [review]

**Pull request merged**